lib/main.js

```diff
@@ -20,29 +20,40 @@ class PHPLanguageClient extends AutoLanguageClient {
     this.statusElement.className = 'inline-block'
   }
 
+  getConnectionType() {
+    const configConnectionType = atom.config.get('ide-php.connectionType')
+    switch (configConnectionType) {
+      case 'auto':    return process.platform === 'win32' ? 'socket' : 'stdio';
+      case 'socket':  return 'socket';
+      case 'stdio':   return 'stdio';
+      default:
+        atom.notifications.addWarning('Invalid connection type setting', {
+          dismissable: true,
+          buttons: [ { text: 'Set Connection Type', onDidClick: () => this.openPackageSettings() } ],
+          description: 'The connection type setting should be set to "auto", "socket" or "stdio". Running in '
```

Comment:
Looks like you didn't finish this description?

Reply:
Thanks, just checked in a fix. Weirdly I did write a whole description, I guess I must have somehow selected the text and overwrote :D